### DISCUSSION TITLE
Suggestion/Question - Can we have file type, tab mode, line ending, etc. in status bar?

### DISCUSSION THREAD

**Discussion by hyler** [zed-industries/zed#11571] [Feature Requests]
Hello,

I was wondering how the devs felt about adding some more status bar info, like the tabs vs spaces mode, line endings, file type, git branch, etc. It's super handy to have it available front and, well, not quite "center" but center enough. It would be so much appreciated.

Something like this:
![image](https://github.com/zed-industries/zed/assets/18198129/7e6518df-bdcf-495f-8abc-87c94882a3dd)

Thank you!

Edit: It would be super duper cool if those were changeable from the status bar, too. (At least the file type [for syntax purposes], the spaces, the encoding, and the line endings.) Example taken from Sublime.

**Comment by AndydeCleyre** [comment]
Absolutely agree to all of this. I'd go even further and suggest moving the breadcrumbs into the status bar.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
